examples/kubernetes/static_provisioning/README.md

```diff
@@ -7,7 +7,7 @@ This example shows how to make a static provisioned Mountpoint for S3 persistent
 - `s3_express_specify_az.yaml` - same as above, but this uses a [S3 Express One Zone](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-one-zone.html) directory bucket and shows how to specify the availability zone (AZ) of the [persistent volume](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) to co-locate the pods with the bucket for lower latency access
```

Comment:
should we add a notice to say we don't support s3 express ? or remove or ~strikethrough~ this line ?

Reply:
Yes we will udpate docs, we have a ticket on that.
The docs need to be re-written completely.
Focusing on most of the code.

Reply:
https://scality.atlassian.net/browse/S3CSI-17